test/datasets/test_fake_dataset.py

```diff
@@ -88,3 +89,9 @@ def test_fake_dataset(with_nan):
     assert isinstance(feat_text_tokenized['input_ids'], MultiNestedTensor)
     assert feat_text_tokenized['input_ids'].dtype == torch.int64
     assert feat_text_tokenized['input_ids'].shape == (num_rows, 2, -1)
+
+    # Test dataset split
+    train_dataset, val_dataset, test_dataset = dataset.split()
+    assert train_dataset.num_rows == num_rows - 2
+    assert val_dataset.num_rows == 1
+    assert test_dataset.num_rows == 1
```

Comment:
Mind doing this in a separate pr? This piece of code seems to be unrelated to text tokenizer.

Reply:
Merge for now to unblock Weihua.